luda-editor/rpc/src/data/cut_update_action.rs

```diff
@@ -53,10 +54,36 @@ pub enum CutUpdateAction {
     DeleteGraphic {
         graphic_index: Uuid,
     },
-    ChangeGraphicOrder {
+    ChangeGraphicOrder(ChangeGraphicOrderAction),
+}
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct ChangeGraphicOrderAction {
+    pub(crate) graphic_index: Uuid,
+    pub(crate) after_graphic_index: Option<Uuid>,
```

Comment:
If this is opened, they can create it directly.
Put private phantom to force them use `new` function.

Reply:
Ah, it's rpc crate!
Ok, my suggestion is pub graphic_index and after_graphic_index, but put phantom to force them use new function for creation.

Reply:
Good suggestion.
I considered following three methods

1. Hide whole struct and expose constructor 
1. Using phantom as you said
1. Using `#[non_exhaustive]`

I thought that `2.` and `3.` are bit of a hack. It would be difficult to someone read this code.
On the other hand, I think `1.` is a very basic method. I think anyone can easily understand this.
So I used `1.`.

But now I realized. phatom_data is generally used for that kind of purpose. Let me try phantom.